static/redfish/v1/$metadata/index.xml

```diff
@@ -3691,4 +3752,8 @@
         <edmx:Include Namespace="OemChassis"/>
         <edmx:Include Namespace="OemChassis.v1_0_0"/>
     </edmx:Reference>
+    <edmx:Reference Uri="/redfish/v1/schema/OemMessage_v1.xml">
```

Comment:
Why did anything else in this file change? 
You ran scripts/update_schemas? 
Did the person before mess something up?

Reply:
i.e. if you remove your changes, is there still changes here?

Reply:
it still seems to happen without the changes too. should I just add the lines that pertain to this oem?

Reply:
No, you might as well fix the file